<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="SelectionMode">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(1, 26, 255);</string>
       </property>
       <property name="title">
        <string>Selection Mode </string>
       </property>
       <widget class="QPushButton" name="user_logout_btn">
        <property name="geometry">
         <rect>
          <x>422</x>
          <y>388</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(255, 0, 0);
color: rgb(0, 0, 127);</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>251</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="user_get_acc_number_btn">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>171</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 127);
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>Get Account Number</string>
         </property>
        </widget>
        <widget class="QPushButton" name="user_view_acc_balance_btn">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>130</y>
           <width>171</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 127);
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>View Account Balance</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>40</y>
          <width>231</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="user_view_trans_history_btn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 127);
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>View Transaction History</string>
         </property>
        </widget>
        <widget class="QPushButton" name="user_make_transaction_btn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 127);
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>Make Transaction</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>270</y>
          <width>281</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="user_transfer_amount_btn">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>211</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 127);
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>Transfer Amount</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="TransactionPage">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(0, 47, 255);</string>
     </property>
     <property name="title">
      <string>Make A Transaction</string>
     </property>
     <widget class="QPushButton" name="user_do_make_transaction_btn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 160, 162);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>371</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Transaction Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="user_transaction_combo_box">
         <property name="styleSheet">
          <string notr="true">color: rgb(233, 207, 255);
background-color: rgb(85, 0, 255);</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Deposite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Withdraw</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>381</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Enter the Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user_make_transaction_edit_line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(254, 254, 254);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="user_cancel_make_transaction_btn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(12, 12, 12);</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TransferAmount">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>451</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 73, 100);</string>
     </property>
     <property name="title">
      <string>Transfer Amount</string>
     </property>
     <widget class="QPushButton" name="user_transfer_do_amount_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(64, 124, 255);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>401</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 206, 239);</string>
         </property>
         <property name="text">
          <string>Receiver Acc number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="transfer_amount_receiver_edit_line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>401</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 208, 255);
color: rgb(17, 17, 17);</string>
         </property>
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="transfer_amount_amount_edit_line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>401</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 208, 255);
color: rgb(17, 17, 17);</string>
         </property>
         <property name="text">
          <string>Signature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="transfer_amount_signature_edit_line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="user_cancel_transfer_amount_btn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>410</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(12, 12, 12);</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TransactionHistory">
    <widget class="QTableWidget" name="user_transaction_history_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>591</width>
       <height>251</height>
      </rect>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="user_transaction_history_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>360</y>
       <width>83</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
background-color: rgb(124, 255, 139);
color: rgb(20, 20, 20);</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
